sport/app/football/views/fragments/leagueSelector.scala.html

```diff
@@ -18,10 +18,10 @@
     <label for="football-leagues" class="football-leagues__label">Choose league: </label>
     <select class="football-leagues__list" name="competitionUrl" id="football-leagues">
         <option value="/football/@pageType">All @pageType</option>
+        @renderOpts("Internationals")
```

Comment:
As mentioned in the PR this also brings up the Nations League, which isn't currently running I believe? So it might not be desired to be shown here - something to check with Gabriel, I guess?